lib/molecules/components/glide-carousel/glideSlide.tsx

```diff
@@ -16,7 +16,7 @@ import React from 'react';
 import { GlideSlideWrapper } from './glide.style';
 
 // Glide Slide wrapper component
-export const GlideSlide = ({ children }: React.PropsWithChildren<Record<string, never>>) => {
+  export const GlideSlide = ({ children }: React.PropsWithChildren<Record<string, never>>) => {
```

Comment:
```suggestion
export const GlideSlide = ({ children }: React.PropsWithChildren<Record<string, never>>) => {
```
Have you tried to format the file afterwards, there is a strange space before export

Reply:
done